### ISSUE TITLE
running a personnalised :guide

### ISSUE THREAD

**Issue by v2belleville** [neo4j-contrib/neo4j-guides#21]
Hello,

I'm preparing an introduction to cypher for which I would like to use a adapted/completed  version of existing guides.

Ideally I would like to run in an Aura hosted db, a guide that I would put on my own cloud.
Since running a guide in Aura seems to require that it is hosted on a Neo4j server, I could have participant install Neo4j Desktop.

But I I have not been able to set the proper parameter in the Neo4j.conf , so that Neo4j browser can read a guide on a distant server.

If I run a Windows localhost server (which is a lot to ask from participant but that could at least get me started writing the guide), I can get Neo4j browser (and not http://browser.graphapp.io/ ) to sort of read a guide served by localhost. The problem is that the layout is bad, because images and videos are not displayed 
![image](https://github.com/neo4j-contrib/neo4j-guides/assets/8779004/0266d5be-bab4-4487-9598-62e7dea17121)

any help on how to run a personalized guide in one of the mentioned configurations would be highly appreciated

**Comment by martinohanlon** [comment]
There is some documentation [about creating custom guides](https://neo4j.com/developer/guide-create-neo4j-browser-guide).

Including a [worked example](https://neo4j.com/developer/guide-create-neo4j-browser-guide/#example-guide)

I think neo4j desktop is "probably" the easiest solution for accessing a custom guide. You can [configure desktop to whitelist domains](https://neo4j.com/developer/guide-create-neo4j-browser-guide/#host-guide). Below is a video walkthrough on configuring desktop.


https://github.com/neo4j-contrib/neo4j-guides/assets/2329501/878a1377-6446-4838-85dd-9d1415888dd3

**Comment by v2belleville** [comment]
thanks Martin for your answer. 

with 
`browser.remote_content_hostname_whitelist=https://www.v2belleville.eu`  (don't use `browser.remote_content_hostname_whitelist=*`

Neo4j desktop browser reaches the guide 
(http://browser.graphapp.io/ triggers a CROSS origin problem - is it correct that there is no client side solution to a cross-origin problem ?)

but there are the same layout problems that I have with localhost : image and videos are not displayed
![image](https://github.com/neo4j-contrib/neo4j-guides/assets/8779004/368d6e1b-0903-4791-8bb7-afb3d2a881b9)

when a webbrowser displays it just fine https://www.v2belleville.eu/neo4j-guides/pandora-papers/pandora-papers.html

any idea about how to fix those layout issues ?

**Comment by martinohanlon** [comment]
Have you tried including the full URL for images?

```
<img class="img-responsive img-thumbnail" src="index_files/paradise-header.jpg">
```

```
<img class="img-responsive img-thumbnail" src="https://www.v2belleville.eu/neo4j-guides/pandora-papers/index_files/paradise-header.jpg">
```

For guides I have written in the past I include an `IMAGE_URL` variable in the asciidoc and referenced that rather than having to reference the full URL on every link - https://github.com/martinohanlon/neo4j-dinosaur-guide/blob/main/guide/dino.adoc?plain=1

**Comment by v2belleville** [comment]
yes ! worked ! and asciidoc does look like a good idea (your dinosaur guide is fun and great to illustrate the use of graph for categorization !)

now, this screenshot I've seen in a presentation makes me believe it is possible to run a guide in the workspace
![guide in workspace](https://github.com/neo4j-contrib/neo4j-guides/assets/8779004/a23cce24-1bd3-4851-aba0-49320604880a)

4 of them can be reached from the top left button, 
![image](https://github.com/neo4j-contrib/neo4j-guides/assets/8779004/aae26084-5b8c-49e9-b3f6-a45264ae0ffa)

but the movie graph that is in the screenshot is not among them.

for people who might be beginners, I would really prefer to use Aura and either https://workspace-preview.neo4j.io or http://browser.graphapp.io/ than desktop install + config

any idea how the movie guide or a personal guide can been run on an Aura db with the workspace (or online browser) ?

**Comment by martinohanlon** [comment]
Good news. 

Unfortunately I dont believe there is a publicly available process for running custom guides in Aura / Workspace. These guides are created and implemented by Neo4j.
